smac/smbo/smbo.py

```diff
@@ -102,7 +103,11 @@ def run(self):
             The best found configuration
         '''
         self.stats.start_timing()
-        self.incumbent = self.initial_design.run()
+        try:
+            self.incumbent = self.initial_design.run()
+        except TAEAbortException:
```

Comment:
This try/except-statement is simply aiming to ensure that an incumbent is always returned. The options to not abort on first-run-crash would rather fit into the tae-runner, I think.
Related to below: What should the expected failure on ABORTs look like?